app/templates/components/view-item.hbs

```diff
@@ -53,13 +53,13 @@
       <span title={{model.value.completeViewClass}}>{{model.value.viewClass}}</span>
     </div>
     <div class="list__cell-helper">
-      {{send-to-console action=sendObjectToConsole param=model.value.objectId}}
+      {{ui/send-to-console action=sendObjectToConsole param=model.value.objectId}}
     </div>
   {{else}}
     --
   {{/if}}
 {{/list.cell}}
 
 {{#list.cell class="list__cell_size_small list__cell_value_numeric"}}
-  <span class="pill pill--not-clickable pill--small js-view-duration">{{ms-to-time model.value.duration}}</span>
```

Comment:
No need for the `not-clickable` class?

Reply:
It didn't seen to have any effect. The class name didn't really describe what it did anyway. Also, I just changed the base `.pill` class to have that "not-clickable" functionality which is just inheriting the parents cursor.